File: examples/python/kernel_linearbyte.py
def linear_byte ():
	print 'LinearByte'

	from sg import sg
	sg('set_features', 'TRAIN', fm_train_byte, 'RAWBYTE')
	sg('set_features', 'TEST', fm_test_byte, 'RAWBYTE')
	sg('set_kernel', 'LINEAR BYTE', 10)
	sg('init_kernel', 'TRAIN')
	km=sg('get_kernel_matrix')

	sg('init_kernel', 'TEST')
	km=sg('get_kernel_matrix')

if __name__=='__main__':
	from numpy import ubyte
	from tools.load import LoadMatrix
	lm=LoadMatrix()
	fm_train_byte=ubyte(lm.load_numbers('../data/fm_train_byte.dat'))
	fm_test_byte=ubyte(lm.load_numbers('../data/fm_test_byte.dat'))
	linear_byte()

File: examples/undocumented/matlab_and_octave/kernel_linearbyte.m
size_cache=10;

addpath('tools');
fm_train_byte=uint8(load_matrix('../data/fm_train_byte.dat'));
fm_test_byte=uint8(load_matrix('../data/fm_test_byte.dat'));

% LinearByte is b0rked
disp('LinearByte');

sg('set_kernel', 'LINEAR', 'BYTE', size_cache);

sg('set_features', 'TRAIN', fm_train_byte, 'RAWBYTE');
%km=sg('get_kernel_matrix', 'TRAIN');
%
%sg('set_features', 'TEST', fm_test_byte, 'RAWBYTE');
%km=sg('get_kernel_matrix', 'TEST');
%
